freeipa/server/master.sls

```diff
@@ -12,7 +12,7 @@ freeipa_server_install:
         --domain {{ server.domain }}
         --hostname {% if server.hostname is defined %}{{ server.hostname }}{% else %}{{ grains['fqdn'] }}{% endif %}
         --ds-password {{ server.ldap.password }}
-        --admin-password {{ server.admin.get('password', server.ldap.password) }}
+        --admin-password {{ server.admin.password }}
```

Comment:
now why, to use if server.admin.password id defined.
we are in 2018, we should treat passwords better. One option here might be update formula to declare what is credentials backend, cmd.run may conditionally read pillar or use sth like https://github.com/hashicorp/envconsul (that has ability to read admin password from vault/consul/etcd ..?